app/views/root/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<%# <h1>Root#index</h1>
+<p>Find me in app/views/root/index.html.erb</p> %>
```

Comment:
- [ ] [OPTIONAL] Let's also remove all unused lines of code here as well so you can have a clean code base.